index.md

```diff
@@ -4,3 +4,11 @@ layout: default
 
 {% include 01-name.md %}
 
+<br>
+
+{% include 02-image.md %}
+ ![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)   
+<br>
+
+{% include 03-links.md %}
+
```

Comment:
Just finished to include the image as well as the documentary designed for the program.